docs/Researcher/cli-reference/runai-submit-dist-pytorch.md

```diff
@@ -42,7 +42,7 @@ runai submit-pytorch --name distributed-job --replicas=2 -g 1 \
 
 > Resources for non-preemptible jobs are guaranteed and will not be reclaimed at any time
```

Comment:
Same here @itayvallach @jasonnovichRunAI

Reply:
Correct, @jasonnovichRunAI  let's remove all occurrences of `--non-preemptible`